packages/Cmfcmf-React-Fiber.package/GRHookUseProvided.extension/instance/getValueAndRemember..st

```diff
@@ -0,0 +1,12 @@
+*Cmfcmf-React-Fiber
```

Comment:
Good point, I'll remove them before merging. Not having them in here for the moment would have required a second branch on the GR repo to be kept in sync with this since Smalltalk isn't happy about loading code when extended classes are missing...